modules/AOBH_BusinessHours/AOBH_BusinessHours.php

```diff
@@ -0,0 +1,158 @@
+<?PHP
+/*********************************************************************************
```

Comment:
Please use the updated license - see here for it https://suitecrm.com/wiki/index.php/Coding_Standards#License